io.github.pragha_music_player.yml

```diff
@@ -0,0 +1,144 @@
+app-id: io.github.pragha_music_player
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pragha
+rename-icon: pragha
+rename-desktop-file: pragha.desktop
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=home
```

Comment:
why not `xdg-music`

Reply:
It let's you select folders. Might be weird if it does not work in `$HOME`?